### PR TITLE
add brief intro to README

### LINKED ISSUE #1: docs and usage example in readme
There are a few proto-doctests in some docstrings but documenter needs to be set up properly

### PR DESCRIPTION
**Pull request by palday**
closes #1 

Copy-pasta from the narrative docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,14 @@
...
+
+This package provides convenient and modular functionality for standardizing
+regression predictors.  Standardizing predictors can increase numerical
+stability of some estimation procedures when the predictors are on very
+different scales or when they are non-orthogonal.  It can also produce more
+interpretable regression models in the presence of interaction terms.
+
+The examples below demonstrate the use of StandardizedPredictors.jl with GLM.jl,
+but they will work with any modeling package that is based on the [StatsModels.jl
+formula](https://juliastats.org/StatsModels.jl/stable/formula/).
```
Comment (kleinschmidt) [inline README.md:16]: Either add the examples or edit to remove references to 'examples below' (maybe reword with pointer to docs)

FWIW I think it would be good to add some examples here too!

Comment (palday) [reply README.md]: I think examples are nice, but we have some nice ones in the docs and they aren't very long, so I just reworded + linked.

**Review by kleinschmidt — APPROVED** [review]
